### ISSUE TITLE
Learning to summarize from human feedback

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#109]
# TL;DR
Document summaries use ROUGE scores to assess the quality of the summaries, but they do not correctly assess the quality of the summaries. In this study, they generate human-preferred summaries by reinforcement learning using human evaluations as the Reward. They outperformed the existing models in human evaluation. Furthermore, it is robust to changes in data domains (Reddit training → CNN/DM evaluation)
![1 Collect human feedback](https://user-images.githubusercontent.com/24524018/93177582-00cb2d00-f76e-11ea-94f7-82ada1fec729.png)

# Paper URL
https://arxiv.org/abs/2009.01325

# Submission Dates(yyyy/mm/dd)
2020/09/02
# Authors and institutions

Nisan Stiennon, Long Ouyang, Jeff Wu, Daniel M. Ziegler, Ryan Lowe, Chelsea Voss, Alec Radford, Dario Amodei, Paul Christiano
- OpenAI

# Methods

# Results

# Comments
